### PR TITLE
lookup symbols in modules provided by internal tools

### PR DESCRIPTION
**Pull request by moosq**

### CODE REVIEW

File: internal/ntt/fs.go [hunk 1]
```diff
@@ -59,3 +61,47 @@ func findFiles(dir string, matcher func(name string) bool) ([]string, error) {
...
+		} else {
+			if finfo.IsDir() {
+				ret = append(ret, realPath)
+			}
+		}
+	}
+	return ret
+}
+
+func FindAuxiliaryTTCN3Files() []string {
```
Comment (5nord) [inline internal/ntt/fs.go:98]: Exported funktions should have a comment for documentation.
```suggestion
// FindAuxiliaryTTCN3Files returns a list of ttcn3 files provided by Nokia internal toolset k3.
func FindAuxiliaryTTCN3Files() []string {
```

File: internal/ntt/fs.go [hunk 2]
```diff
@@ -59,3 +61,47 @@ func findFiles(dir string, matcher func(name string) bool) ([]string, error) {
...
+			}
+			if finfo.IsDir() {
+				ret = append(ret, realPath)
+				pluginDirFound = true
+			}
+		} else {
+			if finfo.IsDir() {
+				ret = append(ret, realPath)
+			}
+		}
```
Comment (5nord) [inline internal/ntt/fs.go:93]: I have some trouble understanding the function of variable `pluginDirFound`. Wouldn't it be valid to just return the first found directory?

```
if info, err := os.Stat(realPath); err == nil {
        if info.IsDir() {
                  return realPath
         }
}
```
If so, the signature of `findAuxiliaryDirectories` should also be changed to `func findAuxilaryDirectory() string`

**Review by 5nord — APPROVED** [review]

File: internal/ntt/fs.go (hunk 2 above)
Comment (moosq) [reply internal/ntt/fs.go:93]: only for the first 3 entries. The last one is a directory with different content

File: internal/ntt/fs.go [hunk 3]
```diff
@@ -78,19 +78,16 @@ func findAuxiliaryDirectories() []string {
 		if finfo, err = os.Stat(realPath); err != nil {
 			continue
 		}
+		if finfo.IsDir() == false {
```
Comment (5nord) [inline internal/ntt/fs.go:81]:
```suggestion
		if !finfo.IsDir() {
```

File: internal/ntt/fs.go [hunk 4]
```diff
@@ -59,3 +61,45 @@ func findFiles(dir string, matcher func(name string) bool) ([]string, error) {
...
+	for idx, realPath := range dirSuffix {
+		var finfo os.FileInfo = nil
+		realPath = path + realPath
+		if finfo, err = os.Stat(realPath); err != nil {
+			continue
+		}
+		if finfo.IsDir() == false {
+			continue
+		}
+		if idx <= 2 {
```
Comment (5nord) [inline internal/ntt/fs.go:84]: Maybe leave a comment why idx <= 2

**Review by 5nord — APPROVED** [review]

**Pull request merged**